app/web/src/components/ProjectCard/ProjectCard.tsx

```diff
@@ -31,11 +32,9 @@ const ProjectCard = ({ title, mainImage, user, Reaction, cadPackage }) => (
       </div>
       <div className="flex items-center mt-1">
         <div className="w-8 h-8 overflow-hidden rounded-full border border-ch-gray-300 shadow">
-          <ImageUploader
-            className=""
-            aspectRatio={1}
-            imageUrl={user?.image}
-            width={50}
+          <ImageFallback
+            imageId={user?.image} // http://res.cloudinary.com/irevdev/image/upload/c_scale,w_50/v1/CadHub/bc7smqwo9qqmrloyf9xr
```

Comment:
Whats the purpose of these comments?

Reply:
Sorry I was debugging an issue, I'll get these removed.